### PR TITLE
Corrects the circle() description

### PR DESCRIPTION
**Pull request by JulioGitLab**
Changes:

 This clarifies that each point on the edge of the circle is half its diameter.

#### PR Checklist

- [ ] `npm run lint` passes
- [x] [Inline documentation] is included / updated
- [ ] [Unit tests] are included / updated

[Inline documentation]: https://github.com/processing/p5.js/blob/main/contributor_docs/inline_documentation.md
[Unit tests]: https://github.com/processing/p5.js/tree/main/contributor_docs#unit-tests

### CODE REVIEW

File: src/core/shape/2d_primitives.js [hunk 1]
```diff
@@ -490,12 +490,10 @@ p5.prototype.ellipse = function(x, y, w, h, detailX) {
...
  *
- * A circle is a round shape defined by the `x`, `y`, and `d`
- * parameters. `x` and `y` set the location of its center. `d` sets its
- * width and height (diameter). Every point on the circle's edge is the
- * same distance, `d`, from its center. See
- * <a href="#/p5/ellipseMode">ellipseMode()</a> for other ways to set
- * its position.
+ * A circle is a round shape defined by the `x`, `y`, and `d` parameters.
+ * `x` and `y` set the location of its center. `d` sets its width and height (diameter).
+ * Every point on the circle's edge is half the distance, `d`, from its center (radius = diameter/2).
```
Comment (nickmcintyre) [inline src/core/shape/2d_primitives.js:495]: Good catch @JulioGitLab! What do you think about the following?

> Every point on the circle's edge is the same distance, `0.5 * d`, from its center. `0.5 * d` (half the diameter) is the circle's radius.

**Review by nickmcintyre — CHANGES_REQUESTED** [review]

File: src/core/shape/2d_primitives.js (hunk 1 above)
Comment (JulioGitLab) [reply src/core/shape/2d_primitives.js:495]: Thanks @nickmcintyre! I think your way of explaining it is easier to understand. Let's use it!

**Review by nickmcintyre — APPROVED** [review]
Looks good to me!

**Comment by Qianqianye** [comment]
Thank you both! @JulioGitLab @nickmcintyre

**Pull request merged**